janus_server/src/message.rs

```diff
@@ -375,8 +386,8 @@ impl Decode for Report {
 /// PPM protocol message representing an arbitrary extension included in a client report.
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct Extension {
-    extension_type: ExtensionType,
-    extension_data: Vec<u8>,
+    pub extension_type: ExtensionType,
+    pub extension_data: Vec<u8>,
```

Comment:
nit:
```suggestion
    pub(crate) extension_type: ExtensionType,
    pub(crate) extension_data: Vec<u8>,
```
It's debatable if being disciplined about this really matters for a binary crate, though we might soon publish a `janus-client` crate.

Reply:
I'm currently touching messages.rs in a different PR that is making a few changes, I'll update this there since ~all of the messages should be modified for consistency.